features/input.yml

```diff
@@ -77,31 +78,14 @@ compat_features:
   # Excluded because of a non-real value
   # - html.elements.input.size
```

Comment:
I think all the non-reals are fixed. Uncomment (or move to appropriate features) and refresh?

Reply:
Per https://github.com/openwebdocs/project/issues/206 it looks like the keys in this PR are all fixed.